src/components/bubble/Bubble.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+
+const Bubble = ({alignment, direction, full, children}) => {
+  let alignmentClass;
+
+  if (['left', 'right'].includes(direction)) {
```

Comment:
I'd put that in a `const` sth like `const VALID_DIRECTIONS = ['left', 'right'];`